src/markdown-pages/terraform/terragrunt-configuration.mdx

```diff
@@ -0,0 +1,578 @@
+	 name = "My Demo Channel"
+	 type = "email"
+
+	 config {
+	 recipients = "your@email_address.com"
+	 include_json_attachment = "1"
+	 }
+	}
+
+	resource "newrelic_alert_policy_channel" "ChannelSubs" {
+	 policy_id = newrelic_alert_policy.DemoPolicy.id
+	 channel_ids = [
+	 newrelic_alert_channel.DemoChannel.id
+	 ]
+	}
+
+	module "HostConditions" {
+	 source = "git::https://github.com/jsbnr/demo-terraform.git"
+	 policyId = newrelic_alert_policy.DemoPolicy.id
+	 cpu_critical = 88
+	 cpu_warning = 78
+	 diskPercent = 68
+	}
+```
+
+You have added a New Relic alert channel, subscribed the DemoPolicy to the alert channel, and added a module hosted on Github.
+
+To clone the module from Github, run `terragrun init` and then run `terragrunt apply`. After Terraform is done processing, You will see your Demo Policy now has two conditions in your New Relic account and one alert channel attached.
+
+</Step>
+
+<!-- ADDING ENVIRONMENT VARIABLE -->
+
+<Step>
+
+## Using your Environment as a Terragrunt Variable
+
+Using Terragrant, you can add the name of the environment you are running to the name of the data you are creating, making your resource more identifiable in New Relic.
+
+In the root __terragrunt.hcl__ file, create a new input for 'env_name'
+```hcl
+inputs = {
+	env_name = "develop"
+}
+```
+
+in the __main.tf__ file add a new varialble for 'env_name'
+```hcl
+variable "env_name" {}
+```
+
+Using string interpolation add the new __env_name__ variable to the alert policy and alert channel resource blocks.
+```hcl
+resource "newrelic_alert_policy" "DemoPolicy" {
+	 name = "${var.env_name}: My Demo Policy"
+	}
+
+resource "newrelic_alert_channel" "DemoChannel" {
+	 name = "${env_name}: My Demo Channel"
+	 type = "email"
+
+	 config {
+	 recipients = "your@email_address.com"
+	 include_json_attachment = "1"
+	 }
+}
+```
+
+Run `terragrunt plan` to see the environment variable added to the configuration name.
+```bash copyable=false
+[output] {muted}# Example output
+[output] ------------------------------------------------------------------------
+[output]
+[output] # newrelic_alert_policy.DemoPolicy will be updated in-place
+[output] ~ resource "newrelic_alert_policy" "DemoPolicy" {
+[output]    id         = "1216533"
+[output]   ~ name        = "My Demo Policy" -> "develop: My Demo Policy"
+[output]    # (2 unchanged attributes hidden)
+[output]  }
+[output]
+[output] # newrelic_alert_policy_channel.ChannelSubs must be replaced
+[output] -/+ resource "newrelic_alert_policy_channel" "ChannelSubs" {
+[output]   ~ channel_ids = [
+[output]     - 4737437,
+[output]    ] -> (known after apply) # forces replacement
+[output]   ~ id     = "1216533:4737437" -> (known after apply)
+[output]    # (1 unchanged attribute hidden)
+[output]  }
+```
+
+Currently, the environment variable name is hardcoded into the root __terragrunt.hcl__ file. You want to avoid having your variables hardcode, using a terragrunt built-in function will pass the value.
```

Comment:
```suggestion
Currently, the environment variable name is hardcoded into the root __terragrunt.hcl__ file. You want to avoid having your variables hardcoded, using a terragrunt built-in function will pass the value.
```

The 2nd sentence here is worded a bit funny. Should that be broken up into 2 sentences instead of separated by a comma?